### PR TITLE
Added sideloading (stable and preview) download sources

### PR DESCRIPTION
**Pull request by TheOnlyTails**
## Description
This PR adds 2 new download sources to the download dropdown in the Hero Section: "Sideload Package" and "Sideload Package (Preview)".
This comes in to replace the "GitHub Release" download source, since now sideloaded packages are served from a separate web server.
Updated packages.

### CODE REVIEW

File: src/data/links.ts [hunk 1]
```diff
@@ -2,9 +2,10 @@ export const links = {
 	discord: "files",
 	twitter: "FilesForWindows",
 	storeId: "9nghp3dx8hdx",
+	sideloadLink: "https://filescommunity.blob.core.windows.net/files/Files.Package.appinstaller",
```
Comment (yaichenbaum) [inline src/data/links.ts:5]:
```suggestion
	sideloadLink: "https://filescommunity.blob.core.windows.net/files/stable/Files.Package.appinstaller",
```
I'm also creating a preview version "https://filescommunity.blob.core.windows.net/files/preview/Files.Package.appinstaller"

**Comment by yaichenbaum** [comment]
@TheOnlyTails can you change the text to say sideload or something similar since it's no longer downloading it fro mGitHub?

**Comment by TheOnlyTails** [comment]
Ah yes, forgot about that.

File: _redirects [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/download/stable  https://ms-appinstaller:?source=https://filescommunity.blob.core.windows.net/files/stable/Files.Package.appinstaller
+/download/preview https://ms-appinstaller:?source=https://filescommunity.blob.core.windows.net/files/preview/Files.Package.appinstaller
```
Comment (yaichenbaum) [inline _redirects:2]:
```suggestion
/download/stable  ms-appinstaller:?source=https://filescommunity.blob.core.windows.net/files/stable/Files.Package.appinstaller
/download/preview ms-appinstaller:?source=https://filescommunity.blob.core.windows.net/files/preview/Files.Package.appinstaller
```

**Comment by TheOnlyTails** [comment]
This won't work, it thinks this is a route like `https://files.community/ms-appinstalller...`

File: src/routes/download/__layout.svelte [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+	import { Button, ProgressRing } from "$lib";
+</script>
+
+<slot/>
+
+<div class="download-page">
+	<h1>Downloading</h1>
+	<ProgressRing size={64}/>
+	<p>Starting download of sideload package...</p>
```
Comment (yaichenbaum) [inline src/routes/download/__layout.svelte:10]: Should this be a thank you message instead? The app installer handless the downloading part.

Comment (TheOnlyTails) [reply src/routes/download/__layout.svelte:10]: Sounds good.

**Review by yaichenbaum — APPROVED** [review]

**Comment by TheOnlyTails** [comment]
I forgot to switch all the checks for the WinGet download source from `currentDownloadSource === 2` to `1`, since it's now the 2nd option, so that's fixed now.

**Pull request merged**